source/install/prod-archlinux.rst

```diff
@@ -0,0 +1,451 @@
+..  _prod-archlinux:
+
+Production Install on Archlinux
+======================================
+
+Install Mattermost in production mode on one, two or three machines.
+
+.. contents::
+    :backlinks: top
+
+
+Set up Database Server
+----------------------
+
+**NOTE**: When Mattermost and postgresql are on the same machine,
+it is recommended to use unix socket mechanism for the connection between Mattermost and Postgresql,
```

Comment:
Propose re-wording for grammar: "...it is recommended that the Unix socket mechanism is used for the connection between..."